assets/js/state/databases.js

```diff
@@ -99,6 +65,34 @@ export const databasesListSlice = createSlice({
         return database;
       });
     },
+    upsertDatabaseInstances: (state, action) => {
+      state.databaseInstances = upsertInstances(
+        state.databaseInstances,
+        action.payload
+      );
+    },
+    removeDatabaseInstance: (state, { payload }) => {
+      state.databaseInstances = state.databaseInstances.filter(
+        (databaseInstance) => !payloadMatchesInstance(databaseInstance, payload)
+      );
+    },
+    updateDatabaseInstanceHealth: (state, { payload }) => {
```

Comment:
Could we rename the parameter from `payload` to `instance` is more understandable, instead of passing pieces of the payload parameter and the parameter itself.

The explanation on the signature of upsert/update instance is clear, but when we use it we could be more explicit

Reply:
You mean to have `updateDatabaseInstanceHealth: (state, { payload: instance }) => {`, right?

Reply:
@CDimonaco Done!
I have taken the chance to rename the `payloadMatchesInstance` function to `instancesMatch`, as it is more meaningful